README.md

```diff
@@ -4,6 +4,11 @@ Powered by [Nextjs](https://github.com/zeit/next.js) and [Tachyons](http://tachy
 
 # Setup
 ## Development
+
```

Comment:
Please move the new instructions under 'Setup' rather than just under development since it's really needed for either.